src/lib/LoginFile.js

```diff
@@ -192,7 +182,7 @@ class LoginFile {
     async _drawDataUrlImage(dataUrl, x, y, w, h) {
         const img = new Image();
         const loaded = new Promise(resolve => {
-            img.onload = () => resolve();
+            img.onload = () => resolve(true);
```

Comment:
I added it because newer Typescript (the version loaded in VSCode) complaints that the Promise is not typed as `void`. I also recently read that creating short arrow functions is better than assigning to browser-native methods, as you never know how their API changes and with what parameters the method will then be called with. So better to define the arguments to use yourself. The `true` really has no effect here, just to shut up TS.